ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/RespondingEthPeer.java

```diff
@@ -218,9 +218,7 @@ public boolean respond(final Responder responder) {
   }
 
   private void respondToMessage(final Responder responder, final OutgoingMessage msg) {
-    boolean supportsRequestId =
-        EthProtocol.isEth66Compatible(msg.capability)
-            && EthProtocol.requestIdCompatible(msg.messageData.getCode());
+    boolean supportsRequestId = EthProtocol.requestIdCompatible(msg.messageData.getCode());
```

Comment:
I thought that all our supported protocols now are request id compatible. Do we still need that method/check?

Reply:
The requestIdCompatible is checking if the message itself requires the requestId. (Some messages don't, such as Status, Transactions)